### PR TITLE
Translate to python sort_events.sh

### PR DESCRIPTION
**Pull request by Daniel-at-github**
Why get rid of the bash script?:

* More platform compatible.
* This is a python centric project.
* I was the cause of this particular bash script.

### CODE REVIEW

**Comment by Daniel-at-github** [comment]
Enhances #32.

**Comment by Daniel-at-github** [comment]
Is the number of feeds types harcoded (event, speaker, tag)? 

I don´t know how to create one more feed, the all events feed where to publish the same list as sort_events.sh but in rss links to events.

File: plugins/writer.py [hunk 1]
```diff
@@ -18,14 +22,31 @@ def sort_elements(self, elements):
...
     @property
     def sorted_events(self):
         if not hasattr(self, '_sorted_events'):
-            output = subprocess.run(["./sort_events.sh"], stdout=subprocess.PIPE)
-            self._sorted_events = output.stdout.decode().strip().split('\n')
+            events = []
+            old_wd = os.getcwd()
+            os.chdir('data')
+            for path in glob.iglob('**/category.json'):
+                directory = os.path.split(path)[0]
```
Comment (zerok) [inline plugins/writer.py]: Here I'd suggest either using `pathlib.Path(path).parent` or `os.path.dirname`.

File: plugins/writer.py [hunk 2]
```diff
@@ -18,14 +22,31 @@ def sort_elements(self, elements):
...
+                command = "git log --diff-filter=A --follow --format=%ai -1 --".split()
+                command.append(path)
+                output = subprocess.check_output(
+                    command, universal_newlines=True)
+                dt = datetime.datetime.strptime(
+                    output.strip(), "%Y-%m-%d %H:%M:%S %z")
+                events.append((dt, directory))
+            os.chdir(old_wd)
+            events.sort(key=operator.itemgetter(1))
```
Comment (zerok) [inline plugins/writer.py:47]: Coud you perhaps combine these into a single sort? Otherwise this looks great, thanks :)

**Comment by zerok** [comment]
Sorry for the late response :(

Regarding creating a custom feed: I haven't seen any options for that anywhere in the Pelican code, but since the URL of the feed will be known, I'd suggest we just integrate it into the PyVideoWriter's write_feed implementation.

File: plugins/writer.py (hunk 2 above)
Comment (Daniel-at-github) [reply plugins/writer.py:47]: I'm afraid It have to be in two steps. In [complex sorts](https://docs.python.org/3.5/howto/sorting.html#sort-stability-and-complex-sorts) you have to sort first the secondary key first.

**Comment by logston** [comment]
Merging as is. Let's make any custom feed changes in a separate PR.

**Comment by Daniel-at-github** [comment]
@logston [pyvideo:sort-feed](https://github.com/pyvideo/pyvideo/tree/sort-feed) branch can't be deleted yet, because last commit didn't make into [pyvideo:master](https://github.com/pyvideo/pyvideo/tree/master).

**Comment by logston** [comment]
Got it. https://github.com/pyvideo/pyvideo/pull/35/

**Pull request merged**